ftw/theming/resources/scss/globals/variables.scss

```diff
@@ -25,6 +25,8 @@ $color-link-hover: $color-secondary !default;
 $color-disabled: $color-gray-light !default;
 $color-text-disabled: $color-gray-dark !default;
 
+$font-family-icon: FontAwesome !default;
```

Comment:
This is only useful for defining icons in a `after` pseudo-element.
Because the font-awesome mixin does not provide a way to attach an icon
to the `after` pseudo-element. This should be reworked later.

I think of a general mixin for attaching icons. The second parameter would be the pseudo element.